routes/accounts.js

```diff
@@ -270,6 +270,44 @@ function accountRoutes (server, options, next) {
     }
   }
 
+  var postAccountsSessionRoute = {
+    method: 'POST',
+    path: '/accounts/{id}/sessions',
+    config: {
+      auth: false,
+      validate: {
+        headers: validations.sessionIdHeader,
+        query: validations.accountQuery,
+        payload: validations.accountPayload,
+        failAction: joiFailAction
+      }
+    },
+    handler: function (request, reply) {
+      var username = request.payload.data.attributes.username
+      var password = request.payload.data.attributes.password
+      var query = request.query
+
+      var sessionId = toSessionId(request)
+
+      admins.validateSession(sessionId)
+
+      .then(function (_) {
```

Comment:
previous examples show this having a parameter `doc` but its not used. Should I do anything with the admin doc before returning the result of the session add?